directives/animate/animate.js

```diff
@@ -1,8 +1,8 @@
-import { directive, PropertyPart } from 'lit/directive.js';
+import { directive, PartType } from 'lit-html/directive.js';
 import { getComposedActiveElement, getNextFocusable } from '../../helpers/focus.js';
-import { AsyncDirective } from 'lit/async-directive.js';
+import { AsyncDirective } from 'lit-html/async-directive.js';
 import { isComposedAncestor } from '../../helpers/dom.js';
-import { noChange } from 'lit';
+import { noChange } from 'lit-html';
```

Comment:
I switched this over to pull everything from `lit-html` or `lit-element`. The `lit` module is currently just wrapping those two. I think what we'll want to do is switch everything in core over at once to reference `lit`.